stdlib/asm/crypto/stark/constants.masm

```diff
@@ -32,475 +32,450 @@ const.NUM_INPUTS_CIRCUIT=232
-#       (FRI_COM_PTR + 128)     ---|
-#              .
-#              .                  | <- Remainder polynomial
-#              .
-#       (FRI_COM_PTR + 256-1)   ---|
-#
-# For each FRI layer, we need 8 memory slots, one for storing the FRI layer commitment and one for
-# storing the word [a0, a1, log2(lde_size), lde_size] where a := (a0, a1) is the folding randomness
-# and lde_size is the size of the LDE domain of the corresponding FRI layer.
-# Since we are using a folding factor of 4 and the maximal degree of the remainder polynomial
-# that we allow is 127, an upper limit of 16 FRI layers is ample and the number of memory slots
-# we thus allocate for this is 128. Moreover, we allocate an additional 128 slots for the remainder
-# polynomial which is expected to be laid out right after the FRI commitments.
+## General
+## Starts at address 3223322624 = 2**31 + 2**30 + 2**21 and the memory region grows forward and is
+## of constant size.
+
+### Addresses to store the LDE domain parameters
+const.LDE_DOMAIN_INFO_PTR=3223322624
+const.LDE_DOMAIN_GEN_PTR=3223322625
+const.LDE_DOMAIN_LOG_SIZE_PTR=3223322626
+const.LDE_DOMAIN_SIZE_PTR=3223322627
+
+### Address to store the number of FRI queries
+const.NUM_QUERIES_PTR=3223322628
+
+### Address to store the size of the FRI remainder polynomial, which is basically the number of its
+### coefficients or equivalently its degree plus one.
+const.REMAINDER_POLY_SIZE_PTR=3223322629
+
+### Address to store the number of FRI folded oracles
+const.NUM_FRI_LAYERS_PTR=3223322630
+
+### The first address of the region of memory storing the FRI remainder polynomial
+const.REMAINDER_POLY_ADDRESS_PTR=3223322631
+
+### Address to store the length of the execution trace
+const.TRACE_LENGTH_PTR=3223322632
+
+### The first address of the region of memory storing the FRI queries, together with some useful
+### data for running FRI check
+const.FRI_QUERIES_ADDRESS_PTR=3223322633
+
+### Address to store the logarithm of the execution trace length
+const.TRACE_LENGTH_LOG_PTR=3223322634
+
+### Address to store the number of grinding bits
+const.GRINDING_FACTOR_PTR=3223322635
+
+### Addresses to store the commitments to main, auxiliary and constraints composition polynomials traces
+const.MAIN_TRACE_COM_PTR=3223322636
+const.AUX_TRACE_COM_PTR=3223322640
+const.COMPOSITION_POLY_COM_PTR=3223322644
+
+### Address to store the OOD evaluation point
+const.Z_PTR=3223322648
+
+### Address to store the zero word, mainly used in the context of RPO
+const.ZERO_WORD_PTR=3223322652
+
+### Address to store the non-deterministically provided DEEP polynomial batching randomness
+const.ALPHA_DEEP_ND_PTR=3223322656
+
+### Address to store the fixed terms, across all queries, of the DEEP queries.
+const.OOD_FIXED_TERM_HORNER_EVALS_PTR=3223322660
+
+### Address to the number of public inputs (in field elements)
+const.NUM_PUBLIC_INPUTS_PTR=3223322664
+
+### Address to trace domain generator
+const.TRACE_DOMAIN_GENERATOR_PTR=3223322668
```

Comment:
RE comment above. The elements referred here are field elements, right? So we don't need to word-align them.